eo-runtime/src/test/eo/org/eolang/memory-tests.eo

```diff
@@ -23,27 +23,34 @@
 +package org.eolang
 +alias org.eolang.io.stdout
 +alias org.eolang.txt.sprintf
++alias org.eolang.hamcrest.assert-that
 +junit
 
 [] > writes-into-memory
   memory > x
-  and. > @
-    x.write 10
-    x.eq 10
+  assert-that > @
+    and.
+      x.write 10
+      x.eq 10
+    $.equal-to TRUE
 
 [] > double-writes
   memory > m
-  and. > @
-    m.write "Hello, world!"
-    m.write 42
-    m.eq 42
+  assert-that > @
+    and.
+      m.write "Hello, world!"
+      m.write 42
+      m.eq 42
+    $.equal-to TRUE
```

Comment:
@Graur the same here. `equal-to TRUE` is basically a cheating, we won't get any meaningful error message if `m` is not equal to 42.